extensions/search-mdn/src/index.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  Detail,
+  Icon,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  useNavigation,
+} from "@raycast/api";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import urljoin from "url-join";
+
+export default function MDNSearchResultsList() {
+  const [query, setQuery] = useState<null | string>(null);
+  const [state, setState] = useState<Result[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const { push } = useNavigation();
+
+  useEffect(() => {
+    async function fetch() {
+      if (query) {
+        setIsLoading(true);
+        const results = await searchMDNByQuery(query);
+        setState(results);
+        setIsLoading(false);
+      }
+    }
+    fetch();
+  }, [query]);
+
+  return (
+    <List
+      isLoading={isLoading}
+      searchBarPlaceholder="Type to search MDN..."
+      onSearchTextChange={(text) => setQuery(text)}
+      throttle
+    >
+      {state.map((result, idx) => (
+        <List.Item
+          id={idx.toString()}
+          key={idx}
+          title={result.title}
+          icon="icon.png"
+          subtitle={result.summary}
+          actions={
+            <ActionPanel>
+              <OpenInBrowserAction url={result.url} />
```

Comment:
It's up to you, but I'd move the "Show Details" action into the primary position (first one) 😉 
And for the Detail keep "Open in Browser" as is.